server/plugin_test.go

```diff
@@ -347,7 +347,7 @@ func TestSyncUsers(t *testing.T) {
 		{
 			Name: "SyncUsers: Unable to store the user info",
 			SetupAPI: func(api *plugintest.API) {
-				api.On("LogError", "Unable to sync user", "error", mock.Anything).Times(1)
+				api.On("LogError", "Unable to set user info during sync user job", "email", "test@test.com", "error", mock.Anything).Times(1)
```

Comment:
Ideally, we shouldn't use mock.Anything here because we can control the error being logged through our mocks.

Reply:
we need to add `: , unable to store the user info` something like this in mock. I think it is fine in this way